account_operating_unit/models/company.py

```diff
@@ -0,0 +1,30 @@
+# © 2016-17 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+from odoo.tools.translate import _
+from odoo.exceptions import UserError
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    inter_ou_clearing_account_id = fields.Many2one('account.account',
+                                                   'Inter-operating unit\
+                                                   clearing account')
+
+    ou_is_self_balanced = fields.Boolean('Operating Units are self-balanced',
+                                         help="Activate if your company is "
```

Comment:
remove trailing space and add in next line consistently

Reply:
@bjeficent Look like this, Same as all fields.
```
ou_is_self_balanced = fields.Boolean(
    'Operating Units are self-balanced',
    help="""Activate if your company is required to generate a balanced
    balance sheet for each operating unit."""
)
```